WLogger/WeekLogs/WeekLogsViewModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  WeekLogsViewModel.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/27/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+
+public class WeekLogsViewModel {
+
+    public let weekTotalTitle: String
+    
+    private let logsAmount = MutableProperty<String>("")
+    public let fetchLogsAmount: Action<AnyObject, String, RepositoryError>
+    
+    private let loggerImage = MutableProperty<UIImage?>(nil)
+    public let fetchLoggerImage: Action<AnyObject, UIImage?, NSError>
+    
+    public let weekLogsTableViewModel: WeekLogsTableViewModel
+    
+    public init(imageFetcher: ImageFetcher, logRepository: LogRepository) {
+        weekTotalTitle = "week total"
+        weekLogsTableViewModel = WeekLogsTableViewModel(logRepository: logRepository)
+
+        fetchLogsAmount = Action { _ in
```

Comment:
Yeah no need for the actions here

```swift
let fetchLoggerImage: SignalProducer<UIImage, ImageFetcherError>
let fetchLogsAmount: SignalProducer<String, RepositoryError>

public init(imageFetcher: ImageFetcher, logRepository: LogRepository) {
    self.fetchLogsAmount =  logRepository.getWeekLogsAmount().observeOn(UIScheduler())
    self.fetchLoggerImage = imageFetcher(FetchLoggerImage).replayLazily(1)
}
```

By using `replayLazily` you are caching the image